pycqed/measurement/waveform_control/pulse.py

```diff
@@ -131,3 +131,10 @@ def chan_wf(self, channel, tvals):
         """
         raise NotImplementedError('chan_wf() not implemented for {}'
                                   .format(str(type(self))[1:-1]))
+
+    @staticmethod
+    def pulse_params():
```

Comment:
I think this should be a classmethod instead. (`@classmethod` with first argument `cls`, similar to `self`)

It would be nice to add this for other existing pulses aswell, such that they can also be used with the device.

Reply:
Done!